### DISCUSSION TITLE
Orientation relationship

### DISCUSSION THREAD

**Discussion by hchaieb1234** [mtex-toolbox/mtex#624] [Ask Anything]
Hello, could you please tell me how to study orientation relationship between two different phases within EBSD data. Is it possible to determine the corresponding axis and angle?

Thank you!

**Comment by lfgmorales** [comment]
https://mtex-toolbox.github.io/Documentation.html Is a good starting point


Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 9 Dec 2020, at 10:18, hchaieb1234 <notifications@github.com> wrote:

﻿

Hello, could you please tell me how to study orientation relationship between two different phases within EBSD data. Is it possible to determine the corresponding axis and angle?

Thank you!

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/624>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN5EKPEBDCXJ72VYTDDST46FPANCNFSM4UTGC63A>.

[ { "@context": "http://schema.org", "@type": "EmailMessage", "potentialAction": { "@type": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/624", "url": "https://github.com/mtex-toolbox/mtex/discussions/624", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { "@type": "Organization", "name": "GitHub", "url": "https://github.com" } } ]
